render/O16_npy_occlude.py

```diff
@@ -22,18 +27,19 @@ def randomCameraPosition(borders):
     camera[x_z_y] = borders[x_z_y][left_right]
     return camera
 
-def occludePointCloud(points, camera, percentage):
+def occludePointCloud(points, new_num_of_points, camera):
     START_RADIUS = 1000 # Hold-over from previous occlusion method; doesn't impact much
     pcd = open3d.geometry.PointCloud()
     pcd.points = open3d.utility.Vector3dVector(points)
     radius = START_RADIUS
-    occlusion = 0
-    while occlusion < percentage:
+    new_points = np.asarray([])
```

Comment:
Needed? You just reassign inside the loop, no?

Reply:
Needed. `new_points` needs to be declared upon entering the loop or else it throws an `UnboundLocalError`. I don't like it either

Reply:
Oh, never mind, I'm an idiot. It's being used in the `while` loop test, so you definitely need it...